handbook/engineering/cloud/index.md

```diff
@@ -6,6 +6,26 @@ The cloud team owns all work that is necessary to build, secure, scale, and oper
 
 1. Any user or organization can use Sourcegraph Cloud with both the private and public code that they care about in a secure way.
 
+## Milestones
+
+1. Any user can add public code
+2. Organizations can use Sourcegraph with up to 50 repos or 100k LoC of private repositories for free
```

Comment:
The "Organizations" here seems pretty vague, how do we define the scope of "an organization"? 

As we allow both users and organizations to own external services, will we have different billing plans for private repos of users and organizations (e.g. GitHub Pro vs GitHub Teams)?

We could probably say the following for now (similar to milestone 4):
```suggestion
2. Any user or organization can use Sourcegraph with up to 50 repos or 100k LoC of private repositories for free
```